packages/apps/plugins/plugin-space/src/components/LimitDialog.tsx

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import React from 'react';
+
+import { Button, Dialog, useTranslation } from '@dxos/react-ui';
+
+import { SPACE_PLUGIN } from '../meta';
+
+export const LimitDialog = () => {
+  const { t } = useTranslation(SPACE_PLUGIN);
+
+  return (
```

Comment:
this is good, but from a longer term perspective is there a way to get away from dialogs and move towards everything being in settings. e.g., status panel in settings that shows a usage bar (and later, actions that can increase your limits). i.e., how can we build incrementally towards where we are going rather than introducing new and temporary UX idioms (that might even be more work than extending what we have).